elastic/kibana
# Remove legacy risk score dashboards when install risk engine

## Remove legacy risk score dashboards when install risk engine

### How to test

- Start kibana
- Create a filebeat index (to have access to host/user pages)
```
PUT filebeat-8.10.0
{
  "mappings": {
    "properties": {
      "@timestamp": {
        "type": "date"
      },
      "host": {
        "properties": {
          "name": {
            "type":"keyword"
          }
        }
      }
    }
  }
}
```
- Go to Security -> Dashboard -> Entity Analytics
- Enable Host risk score an User risk score
- Go to Security -> Dashboards. Scroll down, and be sure that tags about Host risk score and User risk score are enabled. Observe dashboards related to Host/User risk scores.
<img width="804" alt="Screenshot 2023-08-25 at 17 49 08" src="https://github.com/elastic/kibana/assets/7609147/e3cabaa5-8fed-4646-89f9-957c004bc459">

- Then enable this FF `xpack.securitySolution.enableExperimental: ['riskScoringPersistence','riskScoringRoutesEnabled']`
- Restart Kibana, and go to Security -> Manage -> Entity Risk Score
- Click Start update and then confirm it
-  Go back to dashboards and observe that there are no related to Host/User risk score
<img width="993" alt="Screenshot 2023-08-25 at 17 55 40" src="https://github.com/elastic/kibana/assets/7609147/0ced8167-6f39-46fd-8a5a-2dc977712226">
